src/pages/home/components/LegalModal.scss

```diff
@@ -2,3 +2,7 @@
 .panelRoot {
   height: 100%;
 }
+
+.panelPrivacy {
+  height: 77%;
```

Comment:
Why this number? 🤔

Reply:
.innerModal in modal.scss has 100% height. So, If there is less content it is still displayed without adjusting the height itself. That's why the whitespace occurs.